src/frontend/screens/Library/components/InstallModal/index.tsx

```diff
@@ -242,7 +244,9 @@ export default React.memo(function InstallModal({
             accessCode={accessCode}
           >
             {platformSelection()}
```

Comment:
Hmm yeah we can do that. I don't think these are used anywhere else currently though so it would mainly be for readability which I think is a small improvement.

There is also the principle of least exposure in keeping these components private from others since they are not needed elsewhere so they're easier to refactor.

Ultimately it's stylistic but I think these could potentially be used elsewhere and like keeping this channel logic out of  the generic install modal. good idea

Reply:
updated now

Reply:
ah yeah I probably should have left in the same file, but it's not too bad to break it out into it's own file though. it does keep the code a bit more separated. it is simple to ctrl f and find where a component is being used too. a bit more dx overhead though. performance should be the same because everything is rolled up